packages/amplify_core/lib/src/config/amplify_outputs/amplify_outputs.dart

```diff
@@ -55,6 +55,7 @@ class AmplifyOutputs
 
   /// {@macro amplify_core.amplify_outputs.rest_api_outputs}
   @internal
+  @JsonKey(includeToJson: false)
```

Comment:
in `packages/amplify_datastore/lib/amplify_datastore.dart` we call the `.toJson()` method before calling NativeAmplifyBridge.configure(). having the rest_api included in toJson causes the config to become invalid Gen 2 config.

Reply:
Do Amplify iOS and/or Android consider the config invalid when there are extra key/value pairs? That seems potentially problematic since new key/value pairs would generally be considered non breaking.